### PR TITLE
bugfix #2892 – shipped clean breadcrumbs name

### PR DESCRIPTION
**Pull request by winniepukki**
Fixes scandipwa/scandipwa#2892

### CODE REVIEW

File: packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js [hunk 1]
```diff
@@ -78,9 +78,10 @@ export class Breadcrumb extends PureComponent {
 
     renderName() {
         const { name } = this.props;
+        const cleanName = name.replace('+', ' ');
```
Comment (ejnshtein) [inline packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js:81]: what if there will be more than 1 '+' symbol?

I think you need to use RegExp for this.

File: yarn.lock [hunk 2]
```diff
@@ -13046,6 +13046,13 @@ postcss-logical@^3.0.0:
   dependencies:
     postcss "^7.0.2"
 
+postcss-logical@^4.0.2:
```
Comment (ejnshtein) [inline yarn.lock:13049]: Is this dependency relevant to this PR? :thinking:

**Review by ejnshtein — CHANGES_REQUESTED** [review]
Answer to my comments, please!

File: yarn.lock (hunk 2 above)
Comment (winniepukki) [reply yarn.lock:13049]: This is a generally required improvement, however, untied to this PR

File: packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ export class Breadcrumb extends PureComponent {
 
     renderName() {
         const { name } = this.props;
-        const cleanName = name.replace('+', ' ');
+        const cleanName = name.replace(/([+])/g, ' ');
```
Comment (ejnshtein) [inline packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js:81]: You could do `/\+/g`, that'll do the job too.
Otherwise ok.

**Review by ejnshtein — APPROVED** [review]
